lib/src/main/kotlin/konig/kontext/KonigKontextServer.kt

```diff
@@ -3,21 +3,21 @@ package org.konigsoftware.konig.kontext
 import com.google.protobuf.Message
 import io.grpc.BindableService
 import io.grpc.ServerBuilder
-import konig.kontext.CoroutineKonigKontextKey
-import konig.kontext.KonigKontextGrpcContextServerInterceptor
-import kotlin.coroutines.coroutineContext
+import konig.kontext.KONIG_KONTEXT_GRPC_CONTEXT_KEY
+import konig.kontext.KonigKontextServerInterceptor
 import kotlin.reflect.KClass
 
 interface KonigKontextServer<ContextType : Message> {
     @Suppress("UNCHECKED_CAST")
-    suspend fun konigKontext(): ContextType = coroutineContext[CoroutineKonigKontextKey]?.context as? ContextType
+    suspend fun konigKontext(): ContextType = KONIG_KONTEXT_GRPC_CONTEXT_KEY.get() as? ContextType?
```

Comment:
is this function still used to retrieve the context in the server implementation? if so, is it going to be annoying to use if it returns a nullable?

Reply:
We're still throwing if we can't cast